src/main.rs

```diff
@@ -14,14 +14,16 @@ struct Args {
     trace_file: Option<PathBuf>,
     #[structopt(long = "--print_output")]
     print_output: bool,
+    #[structopt(long = "--entrypoint", default_value = "main")]
+    entrypoint: String,
```

Comment:
Do you think you could make this an optional command? 
As I understand it form the issue #180 the main use case for this feature is testing. Perhaps it'd be best to avoid the user of thinking about the entrypoint if they just want to run a program.

Reply:
From my understanding of Clap, since there is a default value, this is already optional.
You are right the user should not have to specify it.

The use case is testing, but not in a sense of testing Cleopatra, but building a Cairo test runner using cleopatra – just wanted to clarify the need :) –.

Reply:
Oh, sorry, I'm fairly new to clap and I wasn't sure how this worked.
Also, I'm glad our project is of use!